csi/src/main.rs

```diff
@@ -310,9 +303,12 @@ struct CliArgs {
     pub run_as: RunAsRole,
     /// Etcd address
     pub etcd_address_vec: Vec<String>,
+    /// Scheduler extender port
+    pub scheduler_extender_port: u16,
 }
 
 /// Parse command line arguments
+#[allow(clippy::too_many_lines)]
```

Comment:
remove too_many_line lint allow

Reply:
This function is to parse command arguments. With the increasing number of args, this function exceeds the limitation. But it makes no sense to split this function.

Reply:
add notes